modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -148,7 +150,7 @@ JPMS introduces stricter encapsulation, impacting how Mule modules interact with
 
 === Add Missing Code
 
-Because reflection is more restricted in Java 17, API objects now require setters. Previously, API objects and plain old Java objects (POJOs) had default constructors and getters for all properties. Now, API objects and POJOs must also have setters so DataWeave can build outside the connector in the Mule app. 
+Because reflection is more restricted in Java 17, API objects now require setters. Previously, API objects and plain old Java objects (POJOs) had default constructors and getters for all properties. Now, API objects and POJOs must also have setters so DataWeave can build outside the connector in the Mule app. Ensuring that API objects now require setters is crucial because it allows for more flexible and robust data handling. This change facilitates compatibility with Java 17's stricter reflection rules and DataWeave's requirements, making sure your connector can interact seamlessly within Mule applications.
```

Comment:
> is crucial because it allows for more flexible and robust data handling. This change facilitates compatibility..

This is not correct. This is mandatory or else it just won't work. I don't understand what this is trying to convey, but it makes it sound like a recommendation when it's actually a mandate